### PR TITLE
Add recipient to update context data

### PR DESCRIPTION
**Pull request by 14kubo12**
KNJ-8084

### CODE REVIEW

File: pymess/models/common.py [hunk 1]
```diff
@@ -171,15 +170,16 @@ class BaseAbstractTemplate(SmartModel):
...
 
-    def render_text_template(self, text, context_data):
-        context_data = self._update_context_data(context_data)
+    def render_text_template(self, text, context_data, recipient):
+        context_data = self._update_context_data(context_data, recipient)
         return Template(text).render(Context(context_data))
 
-    def render_body(self, context_data):
-        return self.render_text_template(self.get_body(), context_data)
+    def render_body(self, context_data, recipient=None):
```
Comment (rubickcz) [inline pymess/models/common.py:181]: I think recipient should be mandatory here, or?

File: pymess/models/emails.py [hunk 2]
```diff
@@ -144,8 +144,8 @@ def clean_subject(self, context_data=None):
     def get_subject(self):
         return self.subject
 
-    def render_subject(self, context_data):
-        context_data = self._update_context_data(context_data)
+    def render_subject(self, context_data, recipient=None):
```
Comment (rubickcz) [inline pymess/models/emails.py:147]: also here...

File: pymess/models/common.py (hunk 1 above)
Comment (14kubo12) [reply pymess/models/common.py:181]: In **clean_body** recipient does not exist, so it is optional

File: pymess/models/emails.py (hunk 2 above)
Comment (14kubo12) [reply pymess/models/emails.py:147]: also here recipient in **clean_subject** does not exist, so it is optional

**Review by rubickcz — APPROVED** [review]

**Pull request merged**